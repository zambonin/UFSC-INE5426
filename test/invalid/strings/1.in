# empty simple quotes
char a = ''
