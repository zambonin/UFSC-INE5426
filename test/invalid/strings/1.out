[Line 3] syntax error, unexpected NL
